openfisca_us/tests/policy/baseline/gov/states/ny/tax/income/credits/ny_real_property_tax_credit.yaml

```diff
@@ -0,0 +1,8 @@
+- name: Homeowner with $25 rent and $5k income.
+  period: 2022
+  input:
+    employment_income: 5_000
```

Comment:
Could you add a unit test? Or switch this to AGI to make it one?

A unit test covering the rent and elderly cases would be good too.